package.json

```diff
@@ -7,13 +7,15 @@
     "example": "examples"
   },
   "scripts": {
+    "build": "node scripts/reference-tables.js",
     "fix": "npm run lint:es -- --fix && npm run lint:css -- --fix",
-    "lint": "npm run lint:es && npm run lint:css && npm run vnu-jar && npm run lint:spelling",
+    "lint": "npm run lint:es && npm run lint:css && npm run vnu-jar && npm run lint:spelling && npm run lint:build",
```

Comment:
Completely unrelated, but should `npm run vnu-jar` be written as `npm run lint:html` for consistency?

Reply:
Seems reasonable. That command is defined. Can you file a new issue?

Reply:
Opened issue https://github.com/w3c/aria-practices/issues/1367